### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Tarasulko**

### CODE REVIEW

**Review by KatyKuzmenko — APPROVED** [review]
Well done!

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,11 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentPerDay = 40;
+  const amount = days * rentPerDay;
+  const cashback = days >= 3 && days < 7 ? 20 : days >= 7 ? 50 : 0;
```
Comment (Anna-Tkachenko) [inline src/calculateRentalCost.js:33]: Rewrite this logic using if/else

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,20 @@
...
+  let cashback = 0;
+  // const cashback = days >= 3 && days < 7 ? 20 : days >= 7 ? 50 : 0;
+
+  if (days >= 3 && days < 7) {
+    cashback = 20;
+  } else if (days >= 7) {
+    cashback = 50;
+  } else {
+    cashback = 0;
+  }
```
Comment (Anna-Tkachenko) [inline src/calculateRentalCost.js:42]: Last else is redundant here
```suggestion
```

**Review by Anna-Tkachenko — APPROVED** [review]
Good job!👍

**Pull request closed without merging**